### PR TITLE
Fix: Check correct options when passing in BoxAnnotations instance

### PR DESCRIPTION
**Pull request by pramodsum**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @pramodsum has signed the CLA. Thanks for the pull request!

File: src/lib/Fullscreen.js [hunk 1]
```diff
@@ -2,20 +2,6 @@ import EventEmitter from 'events';
...
-     *
-     * @return {Fullscreen} Fullscreen instance
-     */
-    constructor() {
-        super();
-
-        document.addEventListener('webkitfullscreenchange', this.fullscreenchangeHandler);
-        document.addEventListener('mozfullscreenchange', this.fullscreenchangeHandler);
-        document.addEventListener('MSFullscreenChange', this.fullscreenchangeHandler);
-        document.addEventListener('fullscreenchange', this.fullscreenchangeHandler);
```
Comment (JustinHoldstock) [inline src/lib/Fullscreen.js]: What're the reasons for removing these event listeners?

Comment (jeremypress) [reply src/lib/Fullscreen.js]: We were playing around with those on Friday. @DanDeMicco has a PR addressing this soon :)

Comment (pramodsum) [reply src/lib/Fullscreen.js]: Whoops I might have forgotten to properly rebase with this. It's unrelated to my commit. Fixing that now

**Pull request merged**